### PR TITLE
⚠️ CONFLICT! Lineage pull request for: skeleton

### PR DESCRIPTION
**Pull request by felddy**
# Lineage Pull Request: CONFLICT #

DANGER

[Lineage] has created this pull request to incorporate new changes found in an
upstream repository:

Upstream repository: [`https://github.com/cisagov/skeleton-docker.git`](https://github.com/cisagov/skeleton-docker.git)

Check the changes in this pull request to ensure they won't cause issues with
your project.

The `lineage/skeleton` branch has **one or more unresolved merge conflicts**
that you must resolve before merging this pull request!

## How to resolve the conflicts ##

1. Clone the `lineage/skeleton` branch locally:

    ```console
    git clone --single-branch --branch lineage/skeleton git@github.com:cisagov/gophish-docker.git
    ```

1. Review each of the following files looking for [conflict-resolution markers](https://git-scm.com/book/en/v2/Git-Branching-Basic-Branching-and-Merging#_basic_merge_conflicts):

    ```console
    CONTRIBUTING.md
    tests/conftest.py
    tests/container_test.py
    ```

1. Resolve each of the conflicts and `add` your changes to the branch:

    ```console
    git add CONTRIBUTING.md tests/conftest.py tests/container_test.py 
    ```

1. Once all the changes have been added, `commit` and `push` the changes back to
GitHub:

    ```console
    git commit --message="Resolved Lineage conflicts."
    git push
    ```

1. Wait for all the automated tests to pass.

1. Check the "Everything is cool" checkbox below:

    - [x] ✌️ The conflicts in this pull request have been resolved.

1. Mark this draft pull request "Ready for review".

------------

**Note:** *You are seeing this because one of this repository's maintainers has
configured [Lineage] to open pull requests.*

For more information:

🛠 [Lineage] configurations for this project are stored in `.github/lineage.yml`

📚 [Read more about Lineage][Lineage]

[//]: # (lineage:metadata:{&quot;slayed&quot;:true})
[Lineage]: https://github.com/cisagov/action-lineage/ "Lineage GitHub Action"

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -99,17 +99,10 @@ can create and configure the Python virtual environment with these
 commands:
 
 ```console
-<<<<<<< HEAD
 cd gophish-docker
 pyenv virtualenv <python_version_to_use> gophish-docker
 pyenv local gophish-docker
 pip install -r requirements-dev.txt
```
Comment (jsf9k) [inline CONTRIBUTING.md]: Should be `pip install --requirement requirements-dev.txt`.

**Review by jsf9k — CHANGES_REQUESTED** [review]
One wafer-thin change requested!

**Review by jsf9k — APPROVED** [review]

File: docker-compose.yml [hunk 2]
```diff
@@ -16,8 +16,10 @@ secrets:
 services:
   gophish:
     build:
-      # VERSION must be specified on the command line:
-      # e.g., --build-arg VERSION=0.0.1
+      # Note build arguments here are for builds using docker-compose (pytest)
+      # The CI build workflow uses docker and passes the value from bump_version
+      args:
+        - VERSION=0.0.5
```
Comment (dav3r) [inline docker-compose.yml:22]: Do we want to modify `bump_version.sh` so that this version number is also updated?

**Comment by dav3r** [comment]
@felddy - Before we merge this thang, we should pull in the late-breaking changes from https://github.com/cisagov/skeleton-docker/pull/24.

**Review by dav3r — COMMENTED** [review]
@felddy - Before we merge this thang, we should pull in the late-breaking changes from https://github.com/cisagov/skeleton-docker/pull/24.

**Comment by felddy** [comment]
> @felddy - Before we merge this thang, we should pull in the late-breaking changes from [cisagov/skeleton-docker#24](https://github.com/cisagov/skeleton-docker/pull/24).

Lineage will append to the PR.

File: docker-compose.yml (hunk 2 above)
Comment (felddy) [reply docker-compose.yml:22]: See 6cc48c2

**Comment by felddy** [comment]
Boom! 💥

File: docker-compose.yml (hunk 2 above)
Comment (dav3r) [reply docker-compose.yml:22]: Very nice!  Is this something that we should push up to `skeleton-docker`?

**Comment by dav3r** [comment]
> Boom! 💥

Yeah boi!

File: docker-compose.yml (hunk 2 above)
Comment (felddy) [reply docker-compose.yml:22]: Yes.

**Review by dav3r — APPROVED** [review]
Nice and phishy in here now! 🐠

**Pull request merged**